commands/team_e2e_test.go

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/spf13/cobra"
+
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+)
+
+func (s *MmctlE2ETestSuite) TestTeamCreateCmdF() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForAllClients("Should not create a team w/o name", func(c client.Client) {
+		printer.Clean()
+		cmd := &cobra.Command{}
+		cmd.Flags().String("display_name", "somedisplayname", "")
+
+		err := createTeamCmdF(c, cmd, []string{})
+		s.EqualError(err, "name is required")
+		s.Require().Empty(printer.GetLines())
+	})
+
+	s.RunForAllClients("Should not create a team w/o display_name", func(c client.Client) {
+		printer.Clean()
+		cmd := &cobra.Command{}
+		cmd.Flags().String("name", model.NewId(), "")
+
+		err := createTeamCmdF(c, cmd, []string{})
+		s.EqualError(err, "display Name is required")
+		s.Require().Empty(printer.GetLines())
+	})
+
+	s.Run("Should create a new team w/ email using LocalClient", func() {
```

Comment:
Im unsure if this one and the next two tests aren't going a little too deep and testing a dependency instead of the CmdF function itself. Penny for your thoughts?

Reply:
Because this is an "e2e" tests, makes sense to test all the way down to something that should be working as expected from the mmctl side. For example, if suddenly this behavior changes on the server, we should update the documentation explaining the expected behavior to the user.

I think we should keep this test cases.